css/form-display.css

```diff
@@ -0,0 +1,47 @@
+@media print {
+    .noprint {
+       visibility: hidden;
+    }
+    @page {
+        size: auto;   /* auto is the initial value */
+        margin: 0;  /* this affects the margin in the printer settings */
+    }
+ }
+
+footer {
+    background-color: #ffffff; 
+    opacity: 100%;
+    width: 189mm; /* Match .styBodyClass width to make this appear on the bottom of the form*/
+    display: flex; 
+    position: fixed; 
+    bottom: 0; 
+    left: 0;
+    align-items: center;
+    margin-left: 0;
+    height: 5em;
+}
+
+button {
+    font-family: "Lato","Helvetica Neue", Helvetica,Arial,sans-serif;
+    font-size: 1rem;
+    font-weight: 700;
+    color: #ffffff;
+    background-color: #1F0C7B;
+    transition: opacity 0.3s ease-out, outline 0.3s ease-out;
+    border: none;
+    border-radius: .5em;
+    cursor: pointer;
+    position: relative;
+    left: 45%;
+}
+
+span.fa {
+    padding-right: 6px;
+}
+
+button:focus {
+    outline: 1px dotted;
+  }
+  button:hover {
+    opacity: 0.7;
+  }
```

Comment:
small thing: a no-space at end of file

Reply:
Maybe this is a prettier question but what's wrong with having no-space at EOF?

Reply:
Yeah this would be handled by prettier. I don't think I've ever once run into an actual problem related to this, but I think without a newline at the end, code added below in the future might appear to edit the last line in the diff.